packages/config/src/forkConfig/index.ts

```diff
@@ -85,22 +85,49 @@ export function createForkConfig(config: ChainConfig): ForkConfig {
   const forksAscendingEpochOrder = Object.values(forks);
   const forksDescendingEpochOrder = Object.values(forks).reverse();
 
+  const blobScheduleDescendingEpochOrder = [...config.BLOB_SCHEDULE].sort((a, b) => b.EPOCH - a.EPOCH);
+
+  const forkBoundariesAscendingEpochOrder: ForkBoundary[] = [
+    // Normal hard-forks (phase0, altair, etc.)
+    ...forksAscendingEpochOrder.map((fork) => ({
+      fork: fork.name,
+      epoch: fork.epoch,
+    })),
+    // Blob Parameter Only (BPO) forks
+    ...config.BLOB_SCHEDULE.map((entry) => ({
+      fork: forksDescendingEpochOrder.find((f) => entry.EPOCH >= f.epoch)?.name ?? phase0.name,
+      epoch: entry.EPOCH,
+    })),
+  ]
+    // Remove unscheduled fork boundaries
```

Comment:
Is it necessary to remove unscheduled boundaries?

Reply:
Probably not, I added it initially because I thought there might be an edge case I am not thinking about which is failing sim tests but I figured out the root cause for that.

We can keep the unscheduled boundaries

Reply:
Let's keep the logic to remove unscheduled fork boundaries. I don't see the usefulness of keeping it if it will never get activated.

It is prone to error when creating and reading fork digest cache (ie. `createCachedGenesis` and `getMetricsTopicStrToLabel`). It causes error in e2e test to fail [here](https://github.com/ChainSafe/lodestar/actions/runs/16087284895/job/45518433585?pr=8022). Instead of improving the two functions, removing unscheduled boundaries is a better fix and create less buggy code